### ISSUE TITLE
does this support a loopback4 app?

### ISSUE THREAD

**Issue by srikanthadibhatla** [mikestaub/serverless-express#31]
loopbak 4 is based on express , is there a way we can use this plugin to proxy the azure function requests to the loopback app?

**Comment by mikestaub** [comment]
I don't think so as this needs to replace the original express instance and I don't think you can do that after it is initialized by loopback.

**Comment by iliasbhal** [comment]
@srikanthadibhatla if it's based on express, you can maybe do:

```
const loopbackApp = require('you/loopback/express/app');
app.use(loopbackApp); 
```
